conda.recipe/bld.bat

```diff
@@ -0,0 +1,11 @@
+npm install .
+npm run build
+"%PYTHON%" setup.py install
+"%PYTHON%" -m nbpresent.install --force --enable
```

Comment:
Technically, this line should be put in a post-script file: http://conda.pydata.org/docs/building/build-scripts.html